portfolio/src/main/webapp/script.js

```diff
@@ -13,16 +13,23 @@
 // limitations under the License.
 
 /**
- * Adds a random voting method to the page.
+ * Adds a random voting method and a nice property it has to the page.
  */
 function addRandomVotingMethod() {
   const votingMethods =
       ['STAR voting', 'Score voting', '3-2-1 voting', 'Approval voting'];
+  const properties =
+      ['Performs well in a variety of simulations making different assumptions',
+       'Simple but expressive',
+       'Highly resistant to strategy',
+       'Incredibly simple, just vote for one or more'];
```

Comment:
I generally find it's difficult to maintain two matching lists. Better to have a list of a class that contains the voting method and property.